ansible/roles/rhacm_assisted_installer/templates/assisted-service-fix.sh

```diff
@@ -0,0 +1,13 @@
+#/usr/bin/env bash
+# set -x
+
+export KUBECONFIG={{ hub_cluster_kubeconfig }}
+node_name=$(oc get po -n assisted-installer -l app=assisted-service -o json | jq -r '.items[].spec.nodeName')
+echo "Node hosting assisted-installer assisted-service: ${node_name}"
+node_ip=$(oc get no ${node_name} -o json | jq -r '.status.addresses[]|select(.type=="InternalIP").address')
+echo "Node: ${node_name}, ip address: ${node_ip}"
+container_id=$(ssh core@${node_ip} sudo crictl ps -a | grep postgres | awk '{print $1}')
+echo "Container ID: ${container_id}"
+host_path=$(ssh core@${node_ip} sudo crictl inspect ${container_id} | jq -r '.status.mounts[]|select(.containerPath=="/var/lib/pgsql/data").hostPath')
+echo "Host path to fix permissions: ${host_path}"
+ssh core@${node_ip} sudo chown 26:26 -R ${host_path}
```

Comment:
Who is uid ``26``? Maybe a non-numeric value will be clearer here

Reply:
Postgres is uid 26.  I will see if we can use the name instead of the uid value.